src/main/app/case/CaseApi.test.ts

```diff
@@ -1,52 +1,59 @@
 import axios from 'axios';
 
-import { CaseApi } from './CaseApi';
+import { CaseApi, getCaseApi } from './CaseApi';
 import { CaseType } from './case';
 
-describe('CaseApi', () => {
-  const results = {
-    data: {
-      id: '1234',
-      error: null,
-      status: 'success',
-      allocatedCourt: {
-        court: 'court',
-      },
-      data: {
-        divorceOrDissolution: 'divorce',
-      },
-    },
-  };
+jest.mock('axios');
+
+const userDetails = {
+  token: '123',
+} as never;
 
-  jest.mock('axios');
+describe('CaseApi', () => {
   const mockedAxios = axios as jest.Mocked<typeof axios>;
 
   const mockLogger = {
     error: async (message: string) => message,
     info: async (message: string) => message,
   } as never;
-  const api = new CaseApi(mockedAxios, mockLogger);
+
+  const api = new CaseApi(mockedAxios, userDetails, mockLogger);
 
   test('Should return case data response', async () => {
-    mockedAxios.get = jest.fn().mockResolvedValue(results);
+    mockedAxios.get = jest.fn().mockResolvedValue({
```

Comment:
This reassignment of `axios.get` could have unintended side effect for other tests and Jest's automocking setup and teardown steps, as this looks like this is modifying axios globally. 

Jest's recommended way to change the mocked resolved value is to call methods directly on a mocked import:

```suggestion
    mockedAxios.get.mockResolvedValue({
```